recipe/gen_patch_json.py

```diff
@@ -653,12 +653,17 @@ def _gen_new_index(repodata, subdir):
                 i = record['depends'].index('python')
                 record['depends'][i] = 'python >=3.5'
 
-        # pip >=21 requires python >=3.6 but the first build has >=3
-        # https://github.com/conda-forge/pip-feedstock/pull/68
         if record_name == 'pip':
+            # pip >=21 requires python >=3.6 but the first build has >=3
+            # https://github.com/conda-forge/pip-feedstock/pull/68
             if record['version'] == "21.0" and record['build'] == "pyhd8ed1ab_0":
                 i = record['depends'].index('python >=3')
                 record['depends'][i] = 'python >=3.6'
+            # Same issue with pip 22 moving to Python 3.7+
+            # https://github.com/conda-forge/pip-feedstock/pull/88
+            if record['version'] in {"22.0", "22.0.1", "22.0.2"} and record['build'] in {"pyhd8ed1ab_0", "pyhd8ed1ab_0", "pyhd8ed1ab_0"}:
```

Comment:
```suggestion
            if record['version'] in {"22.0", "22.0.1", "22.0.2"} and record['build'] == "pyhd8ed1ab_0"}:
```

Just realized these are all the same.